packages/directory/app/styles/navi-directory/components/dir-table.scss

```diff
@@ -36,12 +36,18 @@ $font-family-thin: 'HelveticaNeue-Thin', 'HelveticaNeueW01-Thin', Helvetica, Ari
     font-size: 14px;
     margin: 5px 0;
     padding-left: 5px;
+
+    .dir-table__cell--actions {
+      .d-icon {
+        visibility: hidden;
+      }
+    }
   }
 
   &__row:hover {
-    .actions {
-      .action {
-        visibility: inherit;
+    .dir-table__cell--actions {
+      .d-icon {
+        visibility: visible !important;
```

Comment:
I am seeing some content shift on hover. Can we remove `padding: 10px 0;` on line 57 and add `height: 42px` to the `&__row {` selector`.